src/test/scala/mesosphere/marathon/core/election/CuratorElectionStreamTest.scala

```diff
@@ -58,9 +58,8 @@ class CuratorElectionStreamTest extends AkkaUnitTest with Inside with ZookeeperS
         override lazy val zkTimeoutDuration = 250.milliseconds
       }
 
-      inside(Try(CuratorElectionStream.newCuratorConnection(conf))) {
-        case Failure(ex) =>
-          ex shouldBe a[UnknownHostException]
+      a[Throwable] shouldBe thrownBy {
```

Comment:
I think it should be `an[UnknownHostException] shouldBe thrownBy`.

Reply:
No, the logic changed. The Zookeeper client logs the UnknownHostException, but it just swallows it. It's kind of frustrating. Not really anything we can do.

Reply:
https://github.com/apache/zookeeper/blob/release-3.4.11/src/java/main/org/apache/zookeeper/client/StaticHostProvider.java#L82

Reply:
The end result is an illegal argument exception is thrown because it thinks nothing was passed.

I suppose we could open up an issue.